api_service.py

```
@@ -185,6 +189,132 @@
+                logger.warning(
+                    "[OAuth Debug] 'code' MISSING from body!"
+                )
+
+        except json.JSONDecodeError:
+            logger.exception("[OAuth Debug] Parse failed")
+            body_preview = body_str[:200]
+            logger.info(
+                f"[OAuth Debug] Raw body (200 chars): "
+                f"{body_preview}"
+            )
+        except (KeyError, AttributeError, TypeError, ValueError) as e:
```

Ruff (0.14.10)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)